src/gallery.tsx

```diff
@@ -167,6 +96,9 @@ export const Gallery: FC<GalleryProps> = ({
           src: original,
           srcset: originalSrcset,
           msrc: thumbnail,
+          // @ts-expect-error: right now sortNodes throws exception if any of ref.current in items is not an Element.
+          // Ideally, such exception should not be thrown from sorting function but check should be done before sorting
+          // TODO: extract exception from sortNodes, add typeguard that ref.current is actually an Element
           element: ref.current,
```

Comment:
Photoswipe does not expect, that `element` can be `null` (it expects `HTMLElement | undefined`)

mb instead of 

```ts
element: ref.current, // HTMLElement | null | undefined
```

 we can add something like this:

```ts
...(ref.current !== null ? { element: ref.current } : {}),
```
or

```ts
...(ref.current ? { element: ref.current } : {}),
```
Probably, Photoswipe will not break, if it gets `null` instead of `undefined`, but providing data of expected types seems to be more safe

Reply:
hmm, `sortNodes` should prevent getting null instead of` HTMLElement`

Reply:
right now at this point it is always an Element since `sortNodes` throws exception if non-element met. but `sortNodes` is not executed at exactly one `Item`, so this might be a problem. 

It comes from the point that user can omit attaching ref it there is only one `Item` (as docs says). We discussed with @irudoy that this decision is incorrect and user should attach ref with one `Item` too because it is implementation detail that we should not open to library user. 

SInce this pr has breaking changes we should publish it as a major version. This way we can remove `Can be omitted if there is only one item in the gallery` about `Item` `ref` as another breaking change, extract checking nodes for nullable from `sortNodes` and check it even if there is only one `Item`.